src/components/Tooltip/Tooltip.tsx

```diff
@@ -43,7 +42,7 @@ export const Tooltip: FC<TooltipProps> = ({
       style={style}
       theme={theme}
       trigger={trigger}
```

Comment:
```suggestion
      style={local.style}
      theme={theme}
      trigger={local.trigger}
```

Use the split result.